images.CI/lint-shebang.ps1

```diff
@@ -0,0 +1,38 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
```

Comment:
Looks like input parameters are not used anywhere. Let's remove them in this case.

Reply:
Thank you, I have fixed this.